src/Casts/TimestampRangeCast.php

```diff
@@ -14,4 +14,16 @@ public function getRangeInstance(array $matches): TimestampRange
     {
         return new TimestampRange($matches[2], $matches[3], $matches[1], $matches[4]);
     }
+
+    /**
+     * @param  $value
+     * @return array
+     */
+    protected function parseStringRange($value): array
+    {
+        $matches = [];
+        preg_match('/([\[(])\"?([^",]*)\"?,\"?([^",]*)\"?([])])/', $value, $matches);
```

Comment:
can we extend this function, but not override it?
something like this should work better for future maitainability:
```php
$withoutQuotes = trim($value) // remove quotes without regexp
return parent::parseStringRange($withoutQuotes)
```
can something like this work at all? i havent been coding in php for a long time, so im not sure how to implement this from the top of my head